front/app/components/UI/UserSelect/index.tsx

```diff
@@ -150,10 +118,12 @@ const UserSelect = ({
         backspaceRemovesValue={false}
         menuShouldScrollIntoView={false}
         isClearable
-        filterOption={filterOption}
+        filterOption={() => true}
```

Comment:
We could probably remove this

Reply:
I tried that and it causes a build error. It expects this type. (We don't have types for React Select.)